config/default.json

```diff
@@ -0,0 +1,19 @@
+{
+	"http": {
+		"port": 1337
+	},
+	"rpc": {
+		"port": 8545
+	},
+	"account": {
+		"address": "0x0066AC7A4608f350BF9a0323D60dDe211Dfb27c0",
+		"password": null,
+		"gasPrice": "0x4F9ACA000"
+	},
```

Comment:
Is the gas price fixed? There could be some issues here.

Reply:
Yes, I'm currently trying to sort some stuff out without altering existing behaviours (and that's what we use on Mainnet).